### ISSUE TITLE
Add support for kind whitelist/blacklist

### ISSUE THREAD

**Issue by dtdannen** [UTXOnly/nostpy-relay#80]
I'd like to use nostpy but a hard requirement is that I can whitelist and blacklist Kinds.

I'm curious how adding such a feature could work, and if the maintainers have thought about this support? 

Some ideas for adding this feature:
- add a new environment variable
- add a new feature for a config or settings file that has settings outside of environment variables

**Comment by UTXOnly** [comment]
Yes, this is a feature in the works! I had it in a previous version that worked with [nostpy-gui](https://github.com/UTXOnly/nostpy-gui).

However, I removed it in favor of web-of-trust filtering and to refactor to meet [NIP-86](https://github.com/nostr-protocol/nips/pull/1325) standards. I had kind of gone off on my own and was using notes to manage those lists.

I have the database logic in a separate local dev branch but was waiting until I added the admin API/web GUI (which is also almost complete) to push that feature up.

I will push up the whitelist/blacklist feature hopefully by this weekend. I would love to get your feedback on that before I release everything at once.
